[vs]: docker-syncd-vs.gz build constantly fails

<!--
If you are reporting a new issue, make sure that we do not have any duplicates
already open. You can ensure this by searching the issue list for this
repository. If there is a duplicate, please close your issue and add a comment
to the existing issue instead.

If you suspect your issue is a bug, please edit your issue description to
include the BUG REPORT INFORMATION shown below. If you fail to provide this
information within 7 days, we cannot debug your issue and will close it. We
will, however, reopen it if you later provide the information.

For more information about reporting issues, see
https://github.com/Azure/SONiC/wiki#report-issues

---------------------------------------------------
GENERAL SUPPORT INFORMATION
---------------------------------------------------

The GitHub issue tracker is for bug reports and feature requests.
General support can be found at the following locations:

- SONiC Support Forums - https://groups.google.com/forum/#!forum/sonicproject

---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
Use the commands below to provide key information from your environment:
You do NOT have to include this information if this is a FEATURE REQUEST
-->

**Description**
<!--
Briefly describe the problem you are having in a few paragraphs.
-->
docker-syncd-vs.gz build constantly fails because of unmet dependencies:
https://github.com/Azure/sonic-buildimage/blob/master/platform/vs/docker-syncd-vs/Dockerfile.j2#L11
```
RUN apt-get install -f -y iproute2=4.20.0-2 libcap2-bin=1:2.25-2
```

**Steps to reproduce the issue:**
1. git clone https://github.com/Azure/sonic-buildimage.git
2.  git submodule update --recursive --init --progress
3. make target/docker-syncd-vs.gz

**Describe the results you received:**
Build log:
```
13:04:28  Step 9/20 : RUN apt-get install -f -y iproute2=4.20.0-2 libcap2-bin=1:2.25-2
13:04:28   ---> Running in 2c2817a4abd7
13:04:28  Reading package lists...
13:04:28  Building dependency tree...
13:04:28  Reading state information...
13:04:28  libcap2-bin is already the newest version (1:2.25-2).
13:04:28  libcap2-bin set to manually installed.
13:04:28  Suggested packages:
13:04:28    iproute2-doc
13:04:28  Recommended packages:
13:04:28    libatm1
13:04:28  The following packages will be DOWNGRADED:
13:04:28    iproute2
13:04:28  [91mE: Packages were downgraded and -y was used without --allow-downgrades.
13:04:28  [0m0 upgraded, 0 newly installed, 1 downgraded, 0 to remove and 8 not upgraded.
13:04:28  The command '/bin/sh -c apt-get install -f -y iproute2=4.20.0-2 libcap2-bin=1:2.25-2' returned a non-zero code: 100
13:04:28  [  FAIL LOG END  ] [ target/docker-syncd-vs.gz ]
13:04:28  make: *** [slave.mk:717: target/docker-syncd-vs.gz] Error 1
13:04:29  DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.
13:23:48  make[1]: *** [Makefile.work:264: target/sonic-vs.bin] Error 2
13:23:48  make[1]: Leaving directory '/data/johnar/workspace/vs/buildimage-vs-image/sonic-buildimage'
13:23:48  make: *** [Makefile:9: target/sonic-vs.bin] Error 2
```

**Describe the results you expected:**
Build should pass

**Additional information you deem important (e.g. issue happens only occasionally):**

```
user@build-server: sonic-buildimage$ docker run --rm -ti --name cfg-test docker-config-engine-buster bash

root@31c6dc1ea373:/# apt-get update
Hit:1 http://packages.trafficmanager.net/debian/debian buster InRelease
Hit:2 http://debian-archive.trafficmanager.net/debian buster InRelease
Hit:3 http://packages.trafficmanager.net/debian/debian buster-updates InRelease
Hit:4 http://packages.trafficmanager.net/debian/debian buster-backports InRelease
Hit:5 http://debian-archive.trafficmanager.net/debian-security buster/updates InRelease
Hit:6 http://debian-archive.trafficmanager.net/debian buster-backports InRelease
Reading package lists... Done

root@31c6dc1ea373:/# apt-get install -f -y iproute2=4.20.0-2 libcap2-bin=1:2.25-2
Reading package lists... Done
Building dependency tree
Reading state information... Done
libcap2-bin is already the newest version (1:2.25-2).
libcap2-bin set to manually installed.
Suggested packages:
  iproute2-doc
Recommended packages:
  libatm1
The following packages will be DOWNGRADED:
  iproute2
0 upgraded, 0 newly installed, 1 downgraded, 0 to remove and 8 not upgraded.
E: Packages were downgraded and -y was used without --allow-downgrades.

root@31c6dc1ea373:/# echo $?
100

root@31c6dc1ea373:/# apt-cache policy iproute2
iproute2:
  Installed: 4.20.0-2+deb10u1
  Candidate: 4.20.0-2+deb10u1
  Version table:
     5.9.0-1~bpo10+1 100
        100 http://debian-archive.trafficmanager.net/debian buster-backports/main amd64 Packages
 *** 4.20.0-2+deb10u1 500
        500 http://debian-archive.trafficmanager.net/debian buster/main amd64 Packages
        100 /var/lib/dpkg/status
     4.20.0-2 500
        500 http://packages.trafficmanager.net/debian/debian buster/main amd64 Packages

root@31c6dc1ea373:/# apt-cache policy libcap2-bin
libcap2-bin:
  Installed: 1:2.25-2
  Candidate: 1:2.25-2
  Version table:
 *** 1:2.25-2 500
        500 http://debian-archive.trafficmanager.net/debian buster/main amd64 Packages
        500 http://packages.trafficmanager.net/debian/debian buster/main amd64 Packages
        100 /var/lib/dpkg/status
```

**Output of `show version`:**
```
(paste your output here)
```

**Attach debug file `sudo generate_dump`:**
```
(paste your output here)
```

--- a/platform/vs/docker-gbsyncd-vs/Dockerfile.j2
+++ b/platform/vs/docker-gbsyncd-vs/Dockerfile.j2
@@ -8,7 +8,7 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update
 
-RUN apt-get install -f -y iproute2=4.20.0-2 libcap2-bin=1:2.25-2
+RUN apt-get install -f -y iproute2 libcap2-bin
 
 COPY \
 {% for deb in docker_gbsyncd_vs_debs.split(' ') -%}

--- a/platform/vs/docker-syncd-vs/Dockerfile.j2
+++ b/platform/vs/docker-syncd-vs/Dockerfile.j2
@@ -8,7 +8,7 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update
 
-RUN apt-get install -f -y iproute2=4.20.0-2 libcap2-bin=1:2.25-2
+RUN apt-get install -f -y iproute2 libcap2-bin
 
 COPY \
 {% for deb in docker_syncd_vs_debs.split(' ') -%}

